project.py

```diff
@@ -1,9 +1,11 @@
+import os
```

Comment:
l128 we should set the imported date to the current one instead of the hardcoded one.

Reply:
can you explain this a bit more?

Reply:
The "imported" date is added in the `details` of the issue, but I initially hardcoded it with `2022/01/10`
```
body = body + '\n<li><b>imported</b>: 2022/01/10'
```

Reply:
The "imported" date is added in the `details` of the issue, but I initially hardcoded it with `2022/01/10`
```
body = body + '\n<li><b>imported</b>: 2022/01/10'
```

Reply:
Lol, fixed (and tested)